wp-content/plugins/duracelltomi-google-tag-manager/integration/whichbrowser/src/Analyser/Camouflage.php

```
@@ -0,0 +1,308 @@
+                $this->data->device = $device;
+            }
+
+            $this->data->features[] = 'foundDevice';
+        }
+
+        return $this;
+    }
+
+    private function &detectCamouflagedAndroidAsusBrowser($ua)
+    {
+        if (preg_match('/Linux Ventana; [a-z]{2}(?:-[a-z]{2})?; (.+) Build/u', $ua, $match)) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$match'. (undefined)

(UnusedLocalVariable)